scripts/generate-meetup-data.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+var _ = require('lodash');
+var fs = require('fs');
+var targetFileName = 'free-code-camp-events.mock.json';
+var targetPath = 'js/';
+var meetup_api_key = _.get(process, 'env.MEETUP_API_KEY');
```

Comment:
Wouldn't this be an easier set up for people to just place their key in a `meetup_api_key.txt` file that is in the `.gitignore`. Then just have this line be:

```js
var meetup_api_key = fs.readFileSync('meetup_api_key.txt');
```

The current system works, but I think the instructions in the `README.md` are very machine-dependent, and this could simplify this a lot. Newbies trying to update it aren't going to understand the instructions, especially the ZSH stuff (particularly since we have people running Windows). The stuff in the `README` should be easy enough that anyone could follow the instructions on any platform and not be confused.